osmaxx-py/osmaxx/excerptexport/tests/test_conversion_api_client.py

```diff
@@ -120,15 +121,17 @@ def test_download_files(self):
 
         with vcr.use_cassette(cassette_file_location):
             self.api_client.login()
-            self.api_client.create_job(self.extraction_order, callback_host=self.host)
+            self.api_client.create_job(self.extraction_order, protocol=self.protocol, callback_host=self.host)
 
             if cassette_empty:
                 # wait for external service to complete request
                 time.sleep(120)
 
-            success = self.api_client.download_result_files(self.extraction_order)
+            self.api_client._download_result_files(
+                self.extraction_order,
+                job_status=self.api_client.job_status(self.extraction_order)
+            )
```

Comment:
I think we can leave it as is, if it stands in our way anytime later, we may change it then.